kiwi/utils/temporary.py

```diff
@@ -31,10 +31,10 @@ class Temporary:
     **Provides namespace to handle temporary files and directories**
     """
     def __init__(
-        self, dir: str = defaults.TEMP_DIR, prefix: str = ''
+        self, dir: str = '', prefix: str = ''
```

Comment:
As we are on it, using the built-in name "dir" was not a good idea from me as well. Codacy is also complaining about it. Could we rename the parameter e.g ```dir_name``` or ```tmp_dir_name```

Reply:
I realized changing the parameter name will impact some calls, but I think this would be a good change

Reply:
It is a `path` so maybe using it instead of `dir_name`?
Like in description `Provides namespace to handle temporary files and directories`.

Reply:
Fine by me

Reply:
@schaefi I took another look at the file, it looks like a wrapper around `tempfile.NamedTemporaryFile` and `tempfile.TemporaryDirectory`, which is using `dir` arguments in constructor, so it makes sense to leave the older signature too.

Reply:
yes the methods from the tempfile module uses ```dir``` arguments, which we just have to accept as it is